frontend/packages/operator-lifecycle-manager/src/components/catalog-source.tsx

```diff
@@ -249,10 +253,12 @@ export const CreateSubscriptionYAML: React.FC<CreateSubscriptionYAMLProps> = (pr
 
 const tableColumnClasses = [
   classNames('col-lg-3', 'col-md-3', 'col-sm-4', 'col-xs-6'),
-  classNames('col-lg-2', 'col-md-3', 'col-sm-4', 'col-xs-6'),
-  classNames('col-lg-2', 'col-md-3', 'col-sm-4', 'hidden-xs'),
+  classNames('col-lg-1', 'col-md-2', 'col-sm-2', 'hidden-xs'),
```

Comment:
We should move away from the Bootstrap column classes while in the code. This lets us fit more columns in the table. See #3669 for an example.